### ISSUE TITLE
Improve accessibility of metainfo column for screenreaders

### ISSUE THREAD

**Issue by bspekker** [OpenSlides/openslides-client#6370]
**Current behavior**
In several meeting menus the lists contain some meta info with an icon and then the entry.
Example in agenda list:

<img width="180" height="288" alt="Image" src="https://github.com/user-attachments/assets/632add8c-a7f9-45e2-acb6-4f59930b9dfe" />

The screenreader does not read out the icon or what they stand for, but only the entry.

**Wanted behavior**
A screenreader should read out the label for the icon and then the entry.

Example from picture above: 
1. Visibility public
2. Visibility internal. Duration 0:25h
3. Tag Samstag. Visibility public

This functionality should be added for all list views in meetings:

- [x] agenda (visibility, duration, tags)
- [x] motions (category, motion block, tags) 
- [x] election **(Should be  a ToolTip phase)**
- [x] participants (groups, principals, proxy, structure level, participant number)
- [x] files (restriction, public file)

**Additional context**
The list views on the orga level also need work on this but first need to ensure that the metainfo can be navigated to at all in those views (committees, accounts, files). An additional issue is needed for this.

**Comment by Elblinator** [comment]
> files (restriction, public file)

As far as I understand the guideline for aria labels:
Aria labels should only be added if it's a clickable event which is not explained by the content inside 
given that a public file is a) not a clickable evene and b) has already a tooltip I did not add anything for it
